src/Recipe.php

```diff
@@ -70,6 +70,32 @@ public function isContrib(): bool
         return $this->data['is_contrib'] ?? false;
     }
 
+    public static function queueCompare(Recipe $first, Recipe $second): int
+    {
+        static $order = ['symfony/flex' => -2, 'symfony/framework-bundle' => -1];
```

Comment:
Why not put them in a static class var?

Reply:
I'll put array in a `private static $queueOrder`